<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QDialog" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Algorithme sur les graphes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nom du fichier</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="nomFichier"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoad">
         <property name="text">
          <string>Charger</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="btnCreate">
       <property name="text">
        <string>Créer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblType">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnAPropos">
       <property name="minimumSize">
        <size>
         <width>91</width>
         <height>21</height>
        </size>
       </property>
       <property name="text">
        <string>À propos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGraphicsView" name="graphicsViewGraphe"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="btnTarjan">
           <property name="text">
            <string>Tarjan</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOrdonnancement">
           <property name="text">
            <string>Ordonnancement</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDijkstra">
           <property name="text">
            <string>Dijkstra</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnDantzig">
           <property name="text">
            <string>Dantzig</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnKruskal">
           <property name="text">
            <string>Kruskal</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnPrufer">
           <property name="text">
            <string>Prufer</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="btnData">
           <property name="text">
            <string>Données</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Arcs du graphe :</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblArcs">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
